src/about.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="description" content="Update your description here" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<title>About Us</title>
+</head>
+
+<body>
+  <div class="topnav" id="nav">
+  <img class="logo" src="https://i.ibb.co/cr6WTRB/logo.png" alt="Insomnia Cookies logo">    
+  <a href="index.html">Home</a>
+  <a href="about.html" class="active">About Us</a>
+  <a href="mycart.html">My Cart</a>
+  <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+    <i class="fa fa-bars"></i>
+  </a>
```

Comment:
In your HTML, I would use a `<button>` tag here instead - check out https://medium.com/design-code-repository/a-vs-button-b859547cae4d for more information.

For the JavaScript part of this - try using `addEventListener` like we learned in class the other day to listen to the click event and run a function. You'll have a much easier time with this if you do this, and work in the scripts.js file. Plus, with a button for your HTML, you won't need to void the `href` with JavaScript because you don't need to work against the default action of trying to go to a webpage that the `<a>` tag has attached to it.